extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -347,6 +353,15 @@
     // Unleash
     _gaWikiaPush(['send', 'pageview']);
 
+    if (window.ads.context.opts.showAds) {
```

Comment:
Cool, thanks :) It's safer that way, I think. Since both AdEngine and AnalyticsEngine use the same hook to add scripts to head:
* https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L81,
* https://github.com/Wikia/app/blob/dev/extensions/wikia/AnalyticsEngine/AnalyticsProviderGoogleUA.php#L20.

So, there was a chance of a race condition and it was possible that some users would have JS error in their browsers. 

Now, I only wonder if we shouldn't log to Kibana how many such race condition happens. Do we have a Kibana JS client? :smiling_imp: